public/app/features/alerting/unified/types/silence-form.ts

```diff
@@ -1,5 +1,11 @@
-import { Matcher } from 'app/plugins/datasource/alertmanager/types';
 import { TimeZone } from '@grafana/data';
+import { MatcherOperator } from 'app/plugins/datasource/alertmanager/types';
+
+export type MatcherField = {
```

Comment:
```suggestion
export type MatcherFieldValue = {
```

Imho worthwhile to clarify. `Field` is typically a component rather than an interface